integration_tests/dbt_project.yml

```diff
@@ -146,8 +147,6 @@ seeds:
       +column_types:
         engagement_id: "{{ 'int64' if target.type == 'bigquery' else 'bigint' }}"
         _fivetran_synced: timestamp
-        property_hs_createdate: timestamp
```

Comment:
was there a reason this casting was removed for these fields? it wasn't called out in the CHANGELOG.

Reply:
Yeah I had realized that these fields were not existent in the seed data and therefore not necessary. I'll call this out in the Under the Hood section too